Gemfile

```diff
@@ -3,11 +3,7 @@
 source 'https://rubygems.org'
 
 group :development do
-  gem 'debugger', platform: :ruby_19
-  gem 'pry-debugger', platform: :ruby_19
-  gem 'rcov', platform: :ruby_18
-  gem 'ruby-debug', platform: :ruby_18
-  gem 'simplecov', platform: %i[ruby_19 ruby_20]
```

Comment:
do we want to cease code coverage?

Reply:
This was for platform ruby_20 and 19, but we aren't using those rubies.

Reply:
thanks.